README.md

```diff
@@ -71,44 +81,62 @@ To view the value contained in the secret as plain text:
 az keyvault secret show --name "AppSecret" --vault-name "<YourKeyVaultName>"
 ```
 
-This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Write this information down. You need it in a later step.
+This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Copy the output from the previous command to Notepad. You need it in a later step.
```

Comment:
In other samples you use 'file editor', here you use Notepad.  Please be consistent.